api/src/main/java/com/cloud/agent/api/to/VirtualMachineTO.java

```diff
@@ -413,4 +413,9 @@ public DeployAsIsInfoTO getDeployAsIsInfo() {
     public void setDeployAsIsInfo(DeployAsIsInfoTO deployAsIsInfo) {
         this.deployAsIsInfo = deployAsIsInfo;
     }
+
+    @Override
+    public String toString() {
+        return String.format("VM {\"id\": %s, \"name\": \"%s\", \"uuid\": \"%s\", \"type\": \"%s\"}", id, name, uuid, type);
```

Comment:
A "cosmetic" observation/question.

This way it will print: `VM {"id": 123, "name": my VM, "uuid": 73ec..uuid, "type": User}`

Wouldn't the idea be printing the following?
`VM {id: "123", name: "my VM", uuid: "73ec..uuid", type: "User"}`

If the second option is desired then it would need to change to:
`return String.format("VM {id: \"%s\", name: \"%s\", uuid: \"%s\", type: \"%s\"}", id, name, uuid, type);`

Reply:
Good observation, done.